src/inmanta/data/__init__.py

```diff
@@ -5421,9 +5450,7 @@ async def create_for_partial_compile(
                     FROM (
                         -- Undeployables in previous version of the model that are not part of the partial compile.
                         (
-                            SELECT rid
-                            FROM rids_undeployable_base_version AS undepl
-                            WHERE NOT undepl.rid=ANY($9)
+                            SELECT rid FROM rids_undeployable_base_version AS undepl
```

Comment:
Doesn't this introduce a new bug, where we include shared resources even if they are being exported again? I'm not 100% confident off the top of my head what `ConfigurationModel.undeployable` is used for and whether it's bad to overpopulate it like this.